### PR TITLE
WIP add gems and formatting for display of topics and bookmarks

### PR DESCRIPTION
**Pull request by TeriPastorino**
@levicole

### CODE REVIEW

File: app/views/topics/show.html.erb [hunk 1]
```diff
@@ -1,18 +1,60 @@
...
+  <%= link_to "Delete Topic", @topic, method: :delete, data: { confirm: 'Are you sure you want to delete topic?' } %> |
+  <%= link_to "New Bookmark", new_topic_bookmark_path(@topic) %> 
+</div>
+
+
+    
+<div class="container-fluid">
+  <div class="row">
 
+    <div class="col-md-3">
```
Comment (levicole) [inline app/views/topics/show.html.erb]: I would start off with just using an unordered list here.

Comment (TeriPastorino) [inline app/views/topics/show.html.erb]: So I was just thinking ahead based on the wireframe. can you share more of your thought process on the suggestion?

Comment (levicole) [inline app/views/topics/show.html.erb]: Well, I was thinking that we could do the same layout with an unordered list. The layout would be more responsive that way too. But after looking at the bootstrap docs, I think you are on the right track with this. Have you seen this?

http://getbootstrap.com/components/#thumbnails-custom-content

I think that might be kind of cool.

Comment (TeriPastorino) [inline app/views/topics/show.html.erb]: had not seen that, but that looks awesome, let's do that! 

----- Original Message -----

Comment (TeriPastorino) [inline app/views/topics/show.html.erb]: I am in i want to do this bootstrap custom content that you sent. I just spent some time yesterday on some websites i visit inspecting elements to see how they laid out there content and came up with what i put in! 

I saw the thumbnail thing on some other site and figured that is what I would be using. but it never occurred to me (DUH) to go to bootstrap site.

**Pull request merged**